server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -35,19 +35,35 @@ table! {
         min_months_of_stock -> Double,
         approval_status -> Nullable<crate::db_diesel::requisition::requisition_row::RequisitionRowApprovalStatusMapping>,
         linked_requisition_id -> Nullable<Text>,
-        is_sync_update -> Bool,
         program_id -> Nullable<Text>,
         period_id -> Nullable<Text>,
         order_type -> Nullable<Text>,
     }
 }
 
+table! {
+    #[sql_name = "requisition"]
+    requistion_is_sync_update (id) {
+        id -> Text,
+        is_sync_update -> Bool,
+    }
+}
+
 joinable!(requisition -> name (name_id));
 joinable!(requisition -> store (store_id));
 joinable!(requisition -> user_account (user_id));
 joinable!(requisition -> period (period_id));
 joinable!(requisition -> program (program_id));
 
+#[derive(AsChangeset)]
+#[table_name = "requistion_is_sync_update"]
+#[cfg_attr(test, derive(Debug, PartialEq, Queryable))]
+struct RequisitionIsSyncUpdate {
```

Comment:
Could have done without this struct, but it was easier to update and check is_sync_update with this

Reply:
Did without this struct at the end, as it turns, AsChangeset doesn't actually add filter by id on the update query.